src/Sylius/Bundle/ApiBundle/CommandHandler/SendContactRequestHandler.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\CommandHandler;
+
+use Sylius\Bundle\ApiBundle\Command\SendContactRequest;
+use Sylius\Bundle\ShopBundle\EmailManager\ContactEmailManagerInterface;
+use Sylius\Component\Channel\Repository\ChannelRepositoryInterface;
+use Symfony\Component\Messenger\Handler\MessageHandlerInterface;
+use Webmozart\Assert\Assert;
+
+/** experimental */
+final class SendContactRequestHandler implements MessageHandlerInterface
+{
+    public function __construct(
+        private ContactEmailManagerInterface $contactEmailManager,
```

Comment:
```suggestion
        private SenderInterface $emailSender,
```
not to be coupled with Shop